npm run test a
exit 127
